packages/tooltip/src/tooltip.tsx

```diff
@@ -92,12 +92,12 @@ export const Tooltip = React.forwardRef(function Tooltip(
   const hiddenProps = pick(_tooltipProps, ["role", "id"])
 
   /**
-   * If the `label` or `aria-label` is empty, there's no
+   * If the `label` is empty, there's no
    * point showing the tooltip. Let's simply return back the children
    *
    * @see https://github.com/chakra-ui/chakra-ui/issues/601
    */
-  if (!(label || ariaLabel)) {
+  if (!label) {
```

Comment:
When `label` is not given but only the `aria-label`, I get the below tooltip
![image](https://user-images.githubusercontent.com/39694575/86505500-0063fc80-bde3-11ea-991a-3c7bfb461f6c.png)

Fixed it by only checking the label. Since tooltip is rendered when aria-label is given but not label